tools/report-converter/report_converter/converter/output_parser.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+from abc import ABCMeta, abstractmethod
+
+
+def get_next(it):
+    """ Returns the next item from the iterator or return an empty string. """
+    try:
+        return next(it)
+    except StopIteration:
+        return ''
+
+
+class Event(object):
+    """ Represents an event message. """
+
+    def __init__(self, path, line, column, message):
+        self.path = path
+        self.line = line
+        self.column = column
+        self.message = message
+
+    def __eq__(self, other):
+        return self.path == other.path and \
+            self.line == other.line and \
+            self.column == other.column and \
+            self.message == other.message
+
+    def __str__(self):
+        return 'path={0}, line={1}, column={2}, message={3}'.format(
+            self.path, self.line, self.column, self.message)
+
+
+class Message(Event):
+    """ Represents a message with an optional event and fixit messages. """
```

Comment:
Please extend the comment with the information that it can contain notes too and this is a diagnostic section in the plist. Which is one report.